src/components/Reports/ReportsFilters/ReportsFilters.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+const ReportsFilters = ({ domains, domainSelected, pages, changeDomain }) => {
+  return (
+    <header className="report-filters">
+      <div className="report-wrapper">
+        <h3>
+          <FormattedMessage id="reports_filters.title" />
+        </h3>
+        <p>
+          <FormattedMessage id="reports_filters.description" />
+        </p>
+        <form action="#" className="form-filters">
+          <fieldset className="filter">
+            <label htmlFor="domain">
+              <FormattedMessage id="reports_filters.domain" />
+            </label>
+            <span className="dropdown-arrow" />
+            <select
+              onChange={changeDomain}
+              id="domain"
+              className=""
+              name=""
+              disabled={domains.length === 0}
+            >
+              {domains.map((domain, index) => (
+                <option key={index} value={domain.id}>
+                  {domain.name}
+                </option>
+              ))}
+            </select>
+            {domains.length ? (
```

Comment:
Since this section is related to `domainSelected`, what do you think about using `domainSelected` as a flag in place of `domains.length`?

```suggestion
            {domainSelected ? (
```

Reply:
fixed